debian/control

```diff
@@ -26,16 +26,15 @@ Vcs-Browser: https://github.com/endlessm/endless-ndn
 
 Package: python-eos-data-distribution
 Architecture: all
-Depends:
- ${misc:Depends},
- ${python:Depends},
- adduser,
- nfd,
- gir1.2-notify-0.7,
- gir1.2-soup-2.4,
- python-avahi,
- python-dbus,
- python-pyndn,
+Depends: adduser,
```

Comment:
i don't care about the options we use, but they should be documented somewhere...